docs/compat-data.md

```diff
@@ -0,0 +1,55 @@
+---
+id: babel-compat-data
+title: "@babel/compat-data"
+---
+
+This package provides compatibility data for various JavaScript features across different browser and runtime environments. It is used by [`@babel/preset-env`](./preset-env.md) internally to determine the required Babel plugins for given [compilation targets](./options.md#targets). The compat data is extracted from the [compat-table project](https://github.com/compat-table/compat-table).
+
+## Install
+
+```shell npm2yarn
+npm install --save-dev @babel/compat-data
+```
+
+## Usage
+
+### plugins
```

Comment:
I didn't include the `plugin-bugfixes` chapter since we might remove the `bugfixes` option in Babel 8.